client/src/curriculum/modules-list.scss

```diff
@@ -0,0 +1,236 @@
+@use "../ui/vars" as *;
+
+.curriculum-content-container.with-sidebar,
+.curriculum-content-container {
+  .curriculum-content {
+    .modules {
+      input[type="radio"]:not(:checked) ~ a,
+      input[type="radio"]:not(:checked) ~ ol {
+        display: none;
+      }
+    }
+
+    .module-contents {
+      h2 {
+        width: 100%;
+      }
+    }
+
+    ol.modules-list-list {
+      display: grid;
+      grid-template-areas:
+        "a   b   c"
+        "hr  hr  hr"
+        "mod mod mod"
+        "t   t   t";
+
+      grid-template-columns: auto;
+      margin: 0;
+      padding: 0;
+      @media screen and (min-width: $screen-sm) {
+        grid-template-areas:
+          "a   b   c   spacer"
+          "hr  hr  hr  hr"
+          "mod mod mod mod"
+          "t   t   t   t";
+
+        grid-template-columns: auto auto auto 1fr;
+      }
+
+      hr {
+        border: none;
+        border-top: 1px solid var(--text-inactive);
+        grid-area: hr;
+        margin: 0 0 1.5rem;
+        width: 100%;
+      }
+
+      li {
+        display: contents;
+
+        > input:checked + label {
+          color: var(--text-primary);
+
+          &::before {
+            height: 0;
+            position: absolute;
+            transform: translate3d(-0.75rem, 0.75rem, 0);
+            width: 0;
+          }
+        }
+
+        > input:checked:focus-visible + label {
+          outline-color: var(--accent-primary);
+          outline-offset: 1px;
+          outline-style: auto;
+        }
+
+        > label {
+          color: var(--text-inactive);
+          cursor: pointer;
+        }
+
+        &:nth-child(2) {
+          > label,
+          > input {
+            grid-area: a;
+          }
+
+          > input:checked + label::before {
+            content: url("../assets/icons/cur-modules-underline.svg#1");
```

Comment:
Could we give those SVG views more descriptive names, maybe by their position (left/middle/right) or width?